Jenkinsfile

```diff
@@ -99,6 +104,37 @@ node {
         echo 'skipped application import'
       }
     }
+
+    stage('export') {
+      if (params.IMPORT_APP_PACKAGE) {
+        withCredentials([
+          string(credentialsId: params.OPS_CENTER_CREDENTIALS, variable: 'API_KEY'),
+        ]) {
+          withEnv(MAKE_ENV) {
+            sh """
+            rm -rf ${TELE_STATE_DIR} && mkdir -p ${TELE_STATE_DIR}
+            tele logout ${EXTRA_GRAVITY_OPTIONS}
+            tele login ${EXTRA_GRAVITY_OPTIONS} -o ${OPS_URL} --token=${API_KEY}
+            make export"""
+          }
+        }
+      } else {
+        echo 'skipped application export'
+      }
+    }
+
+    stage('upload application image to S3') {
+      if (isProtectedBranch(env.TAG) && params.IMPORT_APP_PACKAGE) {
```

Comment:
I'm not familiar with isProtectedBranch and quick googling & search of our other repos doesn't turn up any hits.  Where is this defined?

Reply:
I added that function in the last commit